src/js/components/stock-movement/modals/RejectRequestModalHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+const RejectRequestModalHeader = ({ id }) => (
+  <div className="header">
+    <h4 className="title">
+      <Translate
+        id="react.autosaveFeatureModal.title.label"
```

Comment:
`react.autosaveFeatureModal.title.label`? You sure about that one?

Reply:
🫣